notebooks/labs37_notebooks/GenerateTweet_LSTM.ipynb

```diff
@@ -0,0 +1,1225 @@
+            "Epoch 107/340\n",
+            "146/146 [==============================] - 32s 216ms/step - loss: 0.2523 - accuracy: 0.9516\n",
+            "Epoch 108/340\n",
+            "146/146 [==============================] - 32s 216ms/step - loss: 0.2492 - accuracy: 0.9535\n",
+            "Epoch 109/340\n",
+            "146/146 [==============================] - 32s 216ms/step - loss: 0.2336 - accuracy: 0.9573\n",
+            "Epoch 110/340\n",
+            "146/146 [==============================] - 32s 216ms/step - loss: 0.2258 - accuracy: 0.9574\n",
+            "Epoch 111/340\n",
+            "146/146 [==============================] - 32s 216ms/step - loss: 0.2301 - accuracy: 0.9562\n",
+            "Epoch 112/340\n",
+            "146/146 [==============================] - 32s 218ms/step - loss: 0.2358 - accuracy: 0.9520\n",
+            "Epoch 113/340\n",
+            "146/146 [==============================] - 32s 219ms/step - loss: 0.2194 - accuracy: 0.9575\n",
+            "Epoch 114/340\n",
+            "146/146 [==============================] - 32s 219ms/step - loss: 0.2002 - accuracy: 0.9628\n",
+            "Epoch 115/340\n",
+            "146/146 [==============================] - 32s 218ms/step - loss: 0.1955 - accuracy: 0.9641\n",
+            "Epoch 116/340\n",
+            "146/146 [==============================] - 32s 217ms/step - loss: 0.2367 - accuracy: 0.9493\n",
+            "Epoch 117/340\n",
+            "146/146 [==============================] - 32s 219ms/step - loss: 0.2180 - accuracy: 0.9559\n",
+            "Epoch 118/340\n",
+            "146/146 [==============================] - 32s 217ms/step - loss: 0.1855 - accuracy: 0.9663\n",
+            "Epoch 119/340\n",
+            "146/146 [==============================] - 32s 217ms/step - loss: 0.1507 - accuracy: 0.9762\n",
+            "Epoch 120/340\n",
+            "146/146 [==============================] - 32s 216ms/step - loss: 0.3319 - accuracy: 0.9113\n",
+            "Epoch 121/340\n",
+            "146/146 [==============================] - 31s 216ms/step - loss: 0.2853 - accuracy: 0.9273\n",
+            "Epoch 122/340\n",
+            "146/146 [==============================] - 31s 215ms/step - loss: 0.1697 - accuracy: 0.9675\n",
+            "Epoch 123/340\n",
+            "146/146 [==============================] - 32s 216ms/step - loss: 0.1104 - accuracy: 0.9859\n",
+            "Epoch 124/340\n",
+            "146/146 [==============================] - 31s 216ms/step - loss: 0.0914 - accuracy: 0.9892\n",
+            "Epoch 125/340\n",
+            "146/146 [==============================] - 32s 216ms/step - loss: 0.0790 - accuracy: 0.9911\n",
+            "Epoch 126/340\n",
+            "146/146 [==============================] - 32s 218ms/step - loss: 0.0712 - accuracy: 0.9923\n",
+            "Epoch 127/340\n",
+            "146/146 [==============================] - 32s 218ms/step - loss: 0.0636 - accuracy: 0.9932\n",
```

Comment:
Do you think this would be a good cutoff point? This training looks awesome, excuse my ignorance but what exactly is this training?

Reply:
The purpose of the model is to generate rank 2 and 3 tweets. At this point I have the model running for 2 different scenarios:
1. on training data which has both 2 and 3 ranked incidents 
2. a model running on label 2 and label 3 separately.
The thought process behind it is to see which would give us better results. However, I suspect that since the data for the two labels is relatively small we may do better on the combined data. 
Do any of you have thoughts on that?

Reply:
Are you going to implement early stopping as a callback so it doesn't run for hundreds of epochs?

Which is more accurate? Training on both Rank 2 & 3 simultaneously or training on each rank separately? I would assume a priori that training on Rank 2 & 3 separately would give us better results for each.

Reply:
As Joshua mentioned, I think implementing an early stop as a callback to keep it from continuing to run a couple hundred epochs is a fantastic idea!  Other than that great accuracy and very clean, easy to read code.  Good job!